push_reference_update.py

```diff
@@ -0,0 +1,128 @@
+"""
+Script file for generating reference updates
+"""
+
+from idseq_pipeline.commands.common import *
+
+local_work_dir = "idseq_pipeline_temp"
+ncbitool_s3_path = "s3://czbiohub-infectious-disease/ncbitool"
+
+
+def main():
+    ##### PARAMETERS TO EDIT #####
+
+    ## To pull NCBI references through ncbitool, which archives and records versions, set URL_PREFIX=''.
+    # To pull the latest files from NCBI without using ncbitool (no version information), set URL_PREFIX=ftp://ftp.ncbi.nlm.nih.gov.
+    os.environ["URL_PREFIX"] = "ftp://ftp.ncbi.nlm.nih.gov"
+
+    ## Instances with adequate resources for making gsnap and rapsearch indexes.
+    # Need to have write access to destination folders.
+    os.environ["RAPSEARCH_SERVER_IP"] = "54.191.193.210"
+    os.environ["GSNAP_SERVER_IP"] = "34.211.67.166"
+
+    ##### COMMANDS #####
+    date_setup()
+    make_gsnap_index()
+    make_rapsearch_index()
+    make_lineage_files()
```

Comment:
I think the lineage stuff doesn't really need to part of this. But I am fine with it right now. Maybe add a TODO: revisit to decide if we want to make lineage as part of the index making process (or its own)